nbconvert/preprocessors/execute.py

```diff
@@ -220,6 +220,20 @@ class ExecutePreprocessor(Preprocessor):
             )
     ).tag(config=True)
 
+    ipython_hist_file = Unicode(
+        default_value=':memory:',
```

Comment:
:+1: We should have probably had this default a long time ago regardless of parallelism concerns.